nimsuggest/nimsuggest.nim

```diff
@@ -88,6 +89,9 @@ type
     msg: string
     sev: Severity
   CachedMsgs = seq[CachedMsg]
+  NimSuggestCapability = enum 
+  #List of currently supported capabilities. So lang servers/ides can iterate over and check for what's enabled
+    nsCon # current NimSuggest supports the `con` commmand
```

Comment:
So instead of an array of strings you're gonna add enum values here that are not used anywhere in the code explicitly? Bad idea.